examples/redux/App.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { cleanup, fireEvent, render } from 'react-native-testing-library';
+import App from './App';
```

Comment:
We usually don't test the root App component. I mean, we can, but that's easy and requires no setup. 
What we want to test is a component that renders _inside_ the App, that implicitly relies on Redux. A component that doesn't have Provider inside, hence we need to provide such in a test.

Reply:
This has been treated. What I did was to create test cases for the different components that use Redux. For each test component, I created an instance of a store I wanted.